app/src/main/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivity.kt

```diff
@@ -125,4 +123,14 @@ class QuestionPlayerActivity :
   override fun dismissConceptCard() {
     questionPlayerActivityPresenter.dismissConceptCard()
   }
+
+  override fun deleteCurrentProgressAndStopSession() {
+    // No progress is saved in training sessions.
+    questionPlayerActivityPresenter.stopTrainingSession()
+  }
+
+  override fun deleteOldestProgressAndStopSession() {
+    // This function is not needed because their is no progress that is being saved in training
+    // sessions.
+  }
```

Comment:
.

Reply:
So I think my concern here is that the training session experience shouldn't even show the flow that results in progress being deleted (i.e. I'd expect the old dialog to show up & neither of these methods to get called).

Reply:
Should I file a new issue that the training session should have a new dialog box.